### PR TITLE
core/config: add block hsm settings

### PR DESCRIPTION
**Pull request by erykwalder**
This allows the core to configure the remote HSM to be used for block
signing.

### CODE REVIEW

File: core/schema.sql [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```
Comment (jbowens) [inline core/schema.sql]: this is a PostgreSQL 9.6 config. we might want to remove it if other people are still using 9.5. or we could just require everyone to use 9.6 now?

**Comment by jbowens** [comment]
lgtm

File: core/schema.sql (hunk 1 above)
Comment (tessr) [inline core/schema.sql]: updating postgres will require re-bundling the windows installer and (i think) mac app.

**Comment by erykwalder** [comment]
updated

**Comment by jbowens** [comment]
sry meant to approve before but forgot to click the button

**Pull request merged**